CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(imu_vn_100)
 
-add_definitions("-std=c++11")
+set(CMAKE_CXX_FLAGS "-std=gnu++11")
```

Comment:
Why was this needed? Can you describe the error you get when having `std=c++11`.